.github/workflows/ci.yml

```diff
@@ -0,0 +1,46 @@
+name: Snuba Checks
```

Comment:
this is just my personal pref, but I always use lower case, I don't work on this repo though :)

I would just name it `ci` (since `Snuba` is kind of redundant here)